test/e2e/util.go

```diff
@@ -1145,7 +1149,7 @@ func RunRC(config RCConfig) error {
 			dumpNodeDebugInfo(config.Client, containerRestartNodes.List())
 			return fmt.Errorf("%d containers failed which is more than allowed %d", failedContainers, maxContainerFailures)
 		}
-		if len(pods) < len(oldPods) || len(pods) > config.Replicas {
+		if len(pods) < len(oldPods) {
 			// This failure mode includes:
```

Comment:
Ah yes, well the plan is to merge this once we have overshooting solved... But yeah I guess clarifying the comment should be done